sktime/forecasting/neuralforecast.py

```diff
@@ -364,4 +369,346 @@ def get_test_params(cls, parameter_set="default"):
         return params
 
 
-__all__ = ["NeuralForecastRNN"]
+class NeuralForecastLSTM(_NeuralForecastAdapter):
+    """NeuralForecast LSTM model.
+
+    Interface to ``neuralforecast.models.LSMT`` [1]_
+    through ``neuralforecast.NeuralForecast`` [2]_,
+    from ``neuralforecast`` [3]_ by Nixtla.
+
+    The Long Short-Term Memory Recurrent Neural Network (LSTM), uses a
+    multilayer LSTM encoder and an MLP decoder
+
+    Parameters
+    ----------
+    freq : str
+    frequency of the data, see available frequencies [4]_ from ``pandas``
```

Comment:
```suggestion
        frequency of the data, see available frequencies [4]_ from ``pandas``
```

This should be indented.